spec/presenters/scope_filters_presenter_spec.rb

```diff
@@ -1,15 +1,20 @@
 require 'rails_helper'
 
 describe ScopeFiltersPresenter, type: :presenter do
-  describe '#path' do
+  describe '#paths' do
     it "extracts the path from an https URL" do
-      presenter = described_class.new(path: 'https://www.gov.uk/some-path')
-      expect(presenter.path).to eq("/some-path")
+      presenter = described_class.new(paths: ['https://www.gov.uk/some-path', 'https://www.gov.uk/other-path'])
+      expect(presenter.paths).to eq('/some-path, /other-path')
     end
 
     it "extracts the path from an http URL" do
-      presenter = described_class.new(path: 'http://www.gov.uk/some-path')
-      expect(presenter.path).to eq("/some-path")
+      presenter = described_class.new(paths: ['http://www.gov.uk/some-path', 'http://www.gov.uk/other-path'])
+      expect(presenter.paths).to eq('/some-path, /other-path')
+    end
+
+    it "extracts multiple paths separated by commas" do
```

Comment:
This test name suggests to me that some parsing of a string value for paths is going to happen, but we're providing an array with two elements.  The previous two tests also provide a multi-element array so I'm not sure what this test is actually trying to do - perhaps it's a hangover from an earlier version of the implementation and it's no longer required?

Reply:
Removed, it was supposed to test that the a string of comma separated paths would be parsed correctly, but we no longer expect a string. `clean_paths` will send us strictly arrays of values, never a string. I've removed this. It was an old test that got changed to pass without reading the description. 🤦‍♀️